<?xml version="1.0" ?><root>
<report><version>2</version><sessionid>dd7262883afed9545e9eece4</sessionid><type>desync</type><sku>ea.maxis.sims4.13.pc</sku><createtime>2020-04-21 15:57:28</createtime><buildsignature>Local.Unknown.Unknown.1.62.67.1520-1.340.000.32.Release</buildsignature><categoryid>zone_manager.py:62</categoryid><desyncid>dd7262883afed9545e9eece4</desyncid><systemconfig/><screenshot/><desyncdata>Exception in &lt;function c_api_zone_init at 0x2B741228&gt;, args: (213088638521067674, 2323227531, 'EP06_EastHollywood_01', &lt;built-in method set_game_time of NoneType object at 0x73733C10&gt;), kwargs: {}&#13;&#10;RuntimeError('Attempting to set _zone_id to 213088638521067674 when its already set 213088638520995199.') (RuntimeError: Attempting to set _zone_id to 213088638521067674 when its already set 213088638520995199.)&#13;&#10;Traceback (most recent call last):&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 157, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 175, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 246, in c_api_zone_init&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\zone_manager.py", line 62, in create_zone&#13;&#10;RuntimeError: Attempting to set _zone_id to 213088638521067674 when its already set 213088638520995199.&#13;&#10;rtim=0</desyncdata></report>
<report><version>2</version><sessionid>dd7262883afed9545e9eece4</sessionid><type>desync</type><sku>ea.maxis.sims4.13.pc</sku><createtime>2020-04-21 15:57:32</createtime><buildsignature>Local.Unknown.Unknown.1.62.67.1520-1.340.000.32.Release</buildsignature><categoryid>indexed_manager.py:222</categoryid><desyncid>dd7262883afed9545e9eece4</desyncid><systemconfig/><screenshot/><desyncdata>Exception in &lt;function c_api_zone_shutdown at 0x2B741420&gt;, args: (213088638521067674, &lt;built-in method zone_shutdown_completed of NoneType object at 0x73733C10&gt;), kwargs: {}&#13;&#10;AttributeError("'NoneType' object has no attribute 'id'") (AttributeError: 'NoneType' object has no attribute 'id')&#13;&#10;Traceback (most recent call last):&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 114, in wrapped&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 110, in finally_wrap&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 157, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 175, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 291, in c_api_zone_shutdown&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\zone_manager.py", line 84, in remove_id&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\indexed_manager.py", line 205, in remove_id&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\indexed_manager.py", line 222, in remove&#13;&#10;AttributeError: 'NoneType' object has no attribute 'id'&#13;&#10;rtim=0</desyncdata></report>
</root>